laboratory/templates/laboratory/report_organizationlaboratory_list.html

```diff
@@ -0,0 +1,124 @@
+{% extends 'base.html' %}
+{% load bootstrap i18n %}
+
+{% block title %}{% trans 'Organization laboratory report' %}{% endblock %}
+
+{% block content %}
+  <div class="page-header"><h1>{% trans 'Organization laboratory report' %}</h1></div>
+
+  <div>
+  
+ <form action="#" method="post">
+        {% csrf_token %}
+        <table>
+        {{ form|bootstrap_horizontal }}
```

Comment:
Maybe you could use bootstrap forms variant 
https://getbootstrap.com/docs/3.3/css/#forms
Or using a grid system 
https://getbootstrap.com/docs/3.3/css/#grid-example-basic
When you use tables set responsive way 
```
<div class="table-responsive">
  <table class="table">
    ...
  </table>
</div>
```